rules-tests/TypeDeclaration/Rector/ClassMethod/StrictStringParamConcatRector/Fixture/concat_param_append_returned.php.inc

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\StrictStringParamConcatRector\Fixture;
+
+class ConcatParamAppendReturned
+{
+    function formatSizeUnits($bytes): string
+    {
+        if ($bytes >= 1_073_741_824) {
+            $bytes = number_format($bytes / 1_073_741_824, 1).' GB';
+        } elseif ($bytes >= 1_048_576) {
+            $bytes = number_format($bytes / 1_048_576, 1).' MB';
+        } elseif ($bytes >= 1024) {
+            $bytes = number_format($bytes / 1024, 1).' KB';
+        } elseif ($bytes > 1) {
+            $bytes .= ' bytes';
+        } elseif (1 === $bytes) {
+            $bytes .= ' byte';
+        } else {
+            $bytes = '0 bytes';
+        }
+
+        return $bytes;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\StrictStringParamConcatRector\Fixture;
+
+class ConcatParamAppendReturned
+{
+    function formatSizeUnits(?string $bytes): string
```

Comment:
@smortexa hm.., the `$bytes` seems should be possibly `float` or `string` if any possible concatted.

Reply:
I think for safer, it should be skipped when used as Arg.

Reply:
Fixed by skip it 👍